src/cluster.c

```diff
@@ -430,6 +430,7 @@ int clusterSaveConfig(int do_fsync) {
     return 0;
 
 err:
+    serverLog(LL_WARNING, "WARNING: Cluster was not able to save the new configuration on disk: %s", strerror(errno));
```

Comment:
Remove "WARNING: " - it is a duplicate of the logging level of LL_WARNING. Maybe scrub the code base for this pattern? I saw a few places where "WARNING: " is used along with LL_WARNING. 

Also I saw some other issues with how strerror is used across the Redis code base

1) strerror is not multi-thread safe. with Redis 6.x supporting IO threads and potentially more threads getting added, it would be good to switch to the MT-safe version, i.e., strerror_r  
2) strerror is not guaranteed to return a non-NULL string. I would suggest wrapping strerror (actually strerror_r) and return a default string when the translation failed, which should be rare but not guaranteed by the contract.